flask_multipass/providers/ldap/providers.py

```diff
@@ -46,7 +46,9 @@ def ldap_settings(self):
     def set_defaults(self):
         self.ldap_settings.setdefault('timeout', 30)
         self.ldap_settings.setdefault('verify_cert', True)
-        self.ldap_settings.setdefault('cert_file', certifi.where() if certifi else None)
+        self.ldap_settings.setdefault('cert_file', certifi.where()
```

Comment:
Ah, you mean that in case we haven't installed `certifi` but still set our `cert_file` path it will fail anyway? Yeah, hadn't thought of that.